pymc3/tests/test_transforms.py

```diff
@@ -93,6 +93,17 @@ def check_jacobian_det(transform, domain,
             actual_ljd(yval),
             computed_ljd(yval), tol)
 
+def test_ordered():
+    check_vector_transform_identity(tr.ordered, Vector(R, 6))
+
+def test_ordered_jacobian_det():
+    check_jacobian_det(tr.ordered, Vector(R, 2),
+                      tt.dvector, np.array([0, 0]), elemwise=True)
+
+def test_ordered_vals():
```

Comment:
Something like:
```python
def test_ordered_vals():
    vals = get_values(tr.ordered, Vector(R, 3),
                      tt.dvector, np.zeros(3))
    close_to_logical(np.diff(vals) >= 0, True, tol)
```
Again, this test shows that the forward and backward is reversed in your current implementation.